libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -207,10 +213,27 @@ - (instancetype)init {
         self.useSnapshotView = YES;
         self.userAgentString = [self defaultUserAgentString];
         [self setupServiceConfiguration];
+        [self enableEncryptedURLCache];
     }
     return self;
 }
 
+- (void)enableEncryptedURLCache {
+    if (![NSURLCache.sharedURLCache isKindOfClass:[SFSDKEncryptedURLCache class]]) {
+        [NSURLCache.sharedURLCache removeAllCachedResponses];
+        SFSDKEncryptedURLCache *encryptedCache = [[SFSDKEncryptedURLCache alloc] initWithMemoryCapacity:kDefaultCacheMemoryCapacity diskCapacity:kDefaultCacheDiskCapacity diskPath:kDefaultCachePath];
```

Comment:
Should we provide a way for the app to pass those values (memory and disk capacity)?

Reply:
The app can change it through sharedURLCache (`NSURLCache.sharedURLCache.memoryCapacity`) but let me know if that's not good enough